keras/layers/core.py

```diff
@@ -1213,3 +1213,32 @@ def get_config(self):
                   'input_length': self.input_length}
         base_config = super(TimeDistributedDense, self).get_config()
         return dict(list(base_config.items()) + list(config.items()))
+
+
+class GradientReversal(Layer):
+    def __init__(self, l, **kwargs):
```

Comment:
Use a variable with a complete name, not `l`.

Reply:
Hi VanushVaswani,
When I run your example code, I got the TypeError below, how can I fix it ?  Thanks!
Traceback (most recent call last):
  File "dann.py", line 220, in <module>
    dann_model = builder.build_dann_model(main_input)
  File "dann.py", line 182, in build_dann_model
    self.grl = GradientReversal(1.0)
TypeError: __init__() takes 1 positional argument but 2 were given